sig-network/annual-report-2021.md

```diff
@@ -0,0 +1,36 @@
+# 2021 SIG network annual report
+
+This report reflects back on CY 2020 and was written in April 2021.
+
+> Note: Each response should have supporting documentation, linked KEPs, graphs, data
+> Note: Err on the side of being descriptive with your responses. End users and others that read this might not know our upstream acronyms, shorthand, or where to find a reference. 
+
+# Operational
+
+- How are you doing with operational tasks in [sig-governance.md](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md)?
+  - Is your [README](./README.md) accurate? 
+  - Have a CONTRIBUTING.md file?
+  - All subprojects correctly mapped and listed in [sigs.yaml](https://github.com/kubernetes/community/blob/master/sig-list.md)?
+  - What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?
+
+- How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?
```

Comment:
We get periodic updates in the bi-weekly "all-hands" meeting, but they are push not pull (i.e., up to the subproject to bring items to the broader group's attention). No formal reporting procedure. 

@dcbw @jayunit100 @bowei could you guys comment on OWNER files for your respective sub-groups?

Reply:
Gateway API OWNERS file is up to date. Also happy to give more regular/formal updates for Gateway API progress.